test/prototype_transforms_kernel_infos.py

```diff
@@ -2100,3 +2118,39 @@ def sample_inputs_convert_dtype_video():
         ),
     ]
 )
+
+
+def sample_inputs_uniform_temporal_subsample_video():
+    for video_loader in make_video_loaders(sizes=["random"], num_frames=[4]):
+        for temporal_dim in [-4, len(video_loader.shape) - 4]:
+            yield ArgsKwargs(video_loader, num_samples=2, temporal_dim=temporal_dim)
+
+
+def reference_uniform_temporal_subsample_video(video, num_samples, temporal_dim=-4):
+    from pytorchvideo.transforms.functional import uniform_temporal_subsample
+
+    return uniform_temporal_subsample(video, num_samples=num_samples, temporal_dim=temporal_dim)
```

Comment:
If there is a good reference function, we can optionally define a `reference_fn`

https://github.com/pytorch/vision/blob/9c112935abe400222cca8f9fbc2d8386e0f25e80/test/prototype_transforms_kernel_infos.py#L45-L49

In our case we have the perfect reference as we want to mirror the function of `pytorchvideo`. We import it lazily here since we shouldn't expect it to be available when this module is collected. I'll explain what else we need to do to skip this test in case we don't have `pytorchvideo` below.